kernel/kernel-default/src/main/java/io/delta/kernel/parquet/ParquetConverters.java

```diff
@@ -530,6 +530,29 @@ public void resizeIfNeeded()
         }
     }
 
+    public static class FileRowIndexColumnConverter
+            extends LongColumnConverter {
```

Comment:
this doesn't need to extend the `LongColumnConverter` given there is no need for a nullability column.

I am ok if we want to address this in followup PRs.

Reply:
I'll create an issue to do this and the below change in a follow-up PR